package.json

```diff
@@ -39,6 +39,8 @@
         "build:cms:site:skippable": "test -f packages/site/cms-site/lib/index.d.ts && echo 'Skipping CMS Site build' || yarn build:cms:site",
         "build:lib": "lerna run build --concurrency=1 --stream --no-private",
         "publish:release": "lerna publish --force-publish --no-private",
+        "publish:version": "lerna version --force-publish --no-private",
+        "version": "yarn install && git stage yarn.lock",
```

Comment:
why stage yarn.lock?

Reply:
Because `lerna version` automatically pushes, and the `yarn install` in the pipeline causes changes to yarn.lock, which causes the pipeline to fail. I therefore update yarn.lock as part of the `lerna version` command (version is a lifecycle script)